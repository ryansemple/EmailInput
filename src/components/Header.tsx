import React from "react";

interface HeaderProps {
	title: string
}

const Header = (props: HeaderProps) => (
	<header className="Header">
		<h1 className="Header_Title">{props.title}</h1>
	</header>     
)

export default Header;