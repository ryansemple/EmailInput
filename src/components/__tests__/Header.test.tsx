import React from "react";
import { render } from "@testing-library/react";
import Header from "../Header";

test("Renders Header component correctly.", () => {
	const { container } = render
	(
		<Header title="Header Title" />
	);

	expect(container).toMatchSnapshot();
});