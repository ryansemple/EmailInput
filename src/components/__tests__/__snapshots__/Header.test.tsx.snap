// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Header component correctly. 1`] = `
<div>
  <header
    class="Header"
  >
    <h1
      class="Header_Title"
    >
      Header Title
    </h1>
  </header>
</div>
`;
